Minor docs improvements for lightlystudio (#186)
lightly-ai/lightly-studio: lightly_studio/docs/docs/index.md

**Example: Loading images from S3**

```py
import lightly_studio as ls

dataset = ls.Dataset.create(name="s3_dataset")
dataset.add_samples_from_path(path="s3://my-bucket/images/")

ls.start_gui()
```

The images remain in S3 and are streamed to the UI when displayed. Make sure your AWS credentials are configured (via environment variables or `~/.aws/credentials`).

**Current Limitations:**

Cloud storage is only supported for image-only datasets using `add_samples_from_path()`. When loading annotated datasets with `add_samples_from_coco()` or `add_samples_from_yolo()`, both images and annotation files must be stored locally.

### Sample

Each sample is a single data instance. The dataset stores references to all samples, allowing you to access, read, or update their attributes individually.

```py
import lightly_studio as ls

dataset = ls.Dataset.load_or_create(name="my_dataset")
dataset.add_samples_from_path(path="path/to/images")

> 

Reviewer: I don't think this is required, as the below is only pseudo code anyway

start the GUI, run queries and perform selections. It holds the connection to the
database file. Every dataset writes its metadata, tags, annotations, captions, and embeddings into a DuckDB file named `lightly_studio.db`.

```py
import lightly_studio as ls

# Different loading options:
dataset = ls.Dataset.create()

# You can load data directly from a folder
dataset.add_samples_from_path(path="local-folder/some-local-data")

# Or you can load more data at a later point (even across sources such as cloud)
dataset.add_samples_from_path(path="local-folder/some-data-not-in-the-cloud-yet")
dataset.add_samples_from_path(path="gcs://my-bucket-2/path/to/more-images/")

# You can also load a dataset from an .db file (default uses the `lightly_studio.db` file in the working directory)
dataset = ls.Dataset.load()
```

To store the DuckDB file elsewhere (for example, on a larger external disk or to maintain isolated projects), configure the database manager before creating/loading any datasets:

```python
from lightly_studio import db_manager

db_manager.connect(db_file="~/lightly_data/my-db-path.db")
```

!!! note

>     Within the `.db` file we try to store all paths as absolute paths. This allows the software to fetch data for visualization even if you move the .db file around.

Reviewer 2: ```suggestion
    Within the `.db` file all paths are stored as absolute paths. This allows the software to fetch data for visualization even if you move the .db file around.
```
